src/middlewares/checkAuth.ts

```diff
@@ -19,7 +19,7 @@ const checkAuth = async (req: any, res: Response, next: NextFunction): Promise<a
   }
 
   jwt.verify(
-    token,
+    authorization,
```

Comment:
Does the token from the header not being prefixed such as with `Bearer` or `JWT`?